lib/content-services/src/lib/common/services/nodes-api.service.ts

```diff
@@ -239,4 +239,26 @@ export class NodesApiService {
 
         return new NodeMetadata(metadata, nodeEntry.entry.nodeType);
     }
+
+    /**
+     * Gets the list of holds assigned to the node.
+     *
+     * @param nodeId ID of the target node
+     * @returns List of assigned holds
+     */
+    getNodeAssignHolds(nodeId: string): Observable<Hold[]> {
+        const queryOptions = Object.assign({
```

Comment:
You don't need to call `Object.assign` as you're just creating an object

Reply:
use `Object.assign` with `options` object